pkg/clients/cloudsql/cloudsql_test.go

```diff
@@ -452,26 +449,6 @@ func TestIsUpToDate(t *testing.T) {
 			},
 			want: want{upToDate: true, isErr: false},
 		},
-		"IsUpToDateIgnoreReferences": {
```

Comment:
I don't think these were actually doing anything anymore, as IsUpToDate doesn't seem to treat references specially.

Reply:
Confirming this is correct as we now do comparison by translating to the cloud provider representation of the resource rather than the crossplane representation